core/app/models/spree/product/scopes.rb

```diff
@@ -173,6 +173,7 @@ def self.property_conditions(property)
 
     # Can't use add_search_scope for this as it needs a default argument
     def self.available(available_on = nil, _currency = nil)
+      Spree::Deprecation.warn("The second currency argument on Product.available has no effect, and is deprecated", caller) if _currency
```

Comment:
Since we are now using `_currency` (even though it's just for the deprecation). The leading underscore should be removed.

Other than that nitpick :+1: